app/models/action.rb

```diff
@@ -2,4 +2,7 @@ class Action < ApplicationRecord
   belongs_to :task
 
   default_scope { order(order: "asc") }
+
+  validates :guidance_summary, presence: true, if: -> { guidance_text.present? }
```

Comment:
I am not sure we need to validate this - I guess it is okay, but feels a bit too much - less code is always better in my head. I generally see validation as a user input thing, which this is not.